### DISCUSSION TITLE
Translation for tags

### DISCUSSION THREAD

**Discussion by junukwon7** [hydro-dev/Hydro#40] [Q&A] [answered]
How can I manage tags?
![image](https://user-images.githubusercontent.com/48399106/104095413-04b12b00-52da-11eb-9c64-b5b7d02dbf9a.png)

**Comment by undefined-moe** [comment, marked as answer]
should be supported in 2.15.16  
also upgrade ui-default to 4.3.55  
then you can change tags in `control panel -> system settings`

**Reply by junukwon7** [reply]
Nice

**Reply by undefined-moe** [reply]
by the way, I've also added a 'Default display language' setting which controls default display language system-wide. (however this might doesn't work for users already registered)
